src/ts/Offer/Create/Inputs.tsx

```diff
@@ -30,6 +30,7 @@ const inputGenerator = (type, key, label) => {
       return (
         <div className="col">
           <FormsyText
+            id={`offer-create-${label}`}
```

Comment:
Update: `react-html-id` doesn't work with server side (it's funny, there are some methods described in readme that aren't implemented and exported), I leave this with hardcoded `id`s.